### ISSUE TITLE
syslog forwarding not working

### ISSUE THREAD

**Issue by vmalaga** [tsuru/bs#9]
Hi

I had configure month ago a syslog forwarder with syslog but now i can't get any tsuru app log on remote syslog

I had reconfigure bs with the new LOG_XXX variables but nothing. Last i had activate debug on bs and had see this error on docker logs

2016/05/17 11:06:59 [DEBUG] [log forwarder] ignored msg syslogparser.LogParts{"client":"[::1]:47604", "hostname":"docker/af33eefd8c90[2326]:", "tag":"", "facility":3, "rawmsg":[]uint8{0x3c, 0x32, 0x37, 0x3e, 0x4d, 0x61, 0x79, 0x20, 0x31, 0x37, 0x20, 0x31, 0x33, 0x3a, 0x30, 0x36, 0x3a, 0x35, 0x39, 0x20, 0x64, 0x6f, 0x63, 0x6b, 0x65, 0x72, 0x2f, 0x61, 0x66, 0x33, 0x33, 0x65, 0x65, 0x66, 0x64, 0x38, 0x63, 0x39, 0x30, 0x5b, 0x32, 0x33, 0x32, 0x36, 0x5d, 0x3a, 0x20, 0x5b, 0x70, 0x69, 0x64, 0x3a, 0x20, 0x39, 0x7c, 0x61, 0x70, 0x70, 0x3a, 0x20, 0x30, 0x7c, 0x72, 0x65, 0x71, 0x3a, 0x20, 0x32, 0x35, 0x37, 0x33, 0x37, 0x35, 0x2f, 0x32, 0x35, 0x37, 0x33, 0x37, 0x35, 0x5d, 0x20, 0x31, 0x37, 0x32, 0x2e, 0x31, 0x36, 0x2e, 0x37, 0x30, 0x2e, 0x31, 0x31, 0x34, 0x20, 0x28, 0x29, 0x20, 0x7b, 0x34, 0x38, 0x20, 0x76, 0x61, 0x72, 0x73, 0x20, 0x69, 0x6e, 0x20, 0x39, 0x32, 0x38, 0x20, 0x62, 0x79, 0x74, 0x65, 0x73, 0x7d, 0x20, 0x5b, 0x54, 0x75, 0x65, 0x20, 0x4d, 0x61, 0x79, 0x20, 0x31, 0x37, 0x20, 0x31, 0x33, 0x3a, 0x30, 0x36, 0x3a, 0x35, 0x39, 0x20, 0x32, 0x30, 0x31, 0x36, 0x5d, 0x20, 0x47, 0x45, 0x54, 0x20, 0x2f, 0x6c, 0x6f, 0x67, 0x2f, 0x70, 0x72, 0x6f, 0x64, 0x75, 0x63, 0x74, 0x2f, 0x30, 0x33, 0x35, 0x31, 0x33, 0x32, 0x34, 0x38, 0x35, 0x3f, 0x5f, 0x3d, 0x35, 0x33, 0x35, 0x61, 0x31, 0x65, 0x33, 0x38, 0x64, 0x39, 0x26, 0x76, 0x3d, 0x73, 0x77, 0x5f, 0x70, 0x72, 0x69, 0x6e, 0x74, 0x26, 0x68, 0x3d, 0x31, 0x38, 0x65, 0x32, 0x35, 0x36, 0x64, 0x32, 0x37, 0x62, 0x39, 0x33, 0x34, 0x39, 0x66, 0x31, 0x30, 0x62, 0x35, 0x30, 0x61, 0x64, 0x34, 0x62, 0x34, 0x61, 0x34, 0x39, 0x66, 0x62, 0x32, 0x65, 0x30, 0x65, 0x34, 0x37, 0x62, 0x31, 0x63, 0x61, 0x20, 0x3d, 0x3e, 0x20, 0x67, 0x65, 0x6e, 0x65, 0x72, 0x61, 0x74, 0x65, 0x64, 0x20, 0x34, 0x33, 0x20, 0x62, 0x79, 0x74, 0x65, 0x73, 0x20, 0x69, 0x6e, 0x20, 0x38, 0x20, 0x6d, 0x73, 0x65, 0x63, 0x73, 0x20, 0x28, 0x48, 0x54, 0x54, 0x50, 0x2f, 0x31, 0x2e, 0x31, 0x20, 0x32, 0x30, 0x30, 0x29, 0x20, 0x37, 0x20, 0x68, 0x65, 0x61, 0x64, 0x65, 0x72, 0x73, 0x20, 0x69, 0x6e, 0x20, 0x32, 0x35, 0x37, 0x20, 0x62, 0x79, 0x74, 0x65, 0x73, 0x20, 0x28, 0x31, 0x20, 0x73, 0x77, 0x69, 0x74, 0x63, 0x68, 0x65, 0x73, 0x20, 0x6f, 0x6e, 0x20, 0x63, 0x6f, 0x72, 0x65, 0x20, 0x30, 0x29}, "timestamp":time.Time{sec:63599087219, nsec:0, loc:(*time.Location)(0xf3a360)}, "content":"9|app: 0|req: 257375/257375] 172.16.70.114 () {48 vars in 928 bytes} [Tue May 17 13:06:59 2016] GET /log/product/035132485?_=535a1e38d9&v=sw_print&h=18e256d27b9349f10b50ad4b4a49fb2e0e47b1ca => generated 43 bytes in 8 msecs (HTTP/1.1 200) 7 headers in 257 bytes (1 switches on core 0)", "priority":27, "severity":3} error to get appname: No such container: docker/af33eefd8c90[2326]:

Every error is about can't get appname. This is my bs env:

``` code
root@tsrmgr:~# tsuru-admin bs-info 
Image: 

Environment Variables [Default]:
+------------------------------+-------------------------+
| Name                         | Value                   |
+------------------------------+-------------------------+
| SYSLOG_FORWARD_ADDRESSES     | udp://172.16.70.66:5140 |
| STATUS_INTERVAL              | 60                      |
| METRICS_INTERVAL             | 60                      |
| METRICS_BACKEND              | logstash                |
| LOG_SYSLOG_FORWARD_ADDRESSES | udp://172.16.70.66:5140 |
| LOG_SYSLOG_TIMEZONE          | Europe/Madrid           |
+------------------------------+-------------------------+

root@tsr01:~# docker version
Client:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Tue Apr 26 23:30:23 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Tue Apr 26 23:30:23 2016
 OS/Arch:      linux/amd64

```

**Comment by guilhermef** [comment]
It's an issue with the docker api.
It will work if you downgrade docker to 1.10.3.

**Comment by tarsisazevedo** [comment]
@cezarsa improve our log parser to support docker log in 1.11.x version and we are testing it to release asap.

**Comment by vmalaga** [comment]
@tarsisazevedo thanks , we are going to downgrade since the next release.

**Comment by andrestc** [comment]
Hi @vmalaga, this was fixed in [78810e8](https://github.com/tsuru/bs/commit/78810e80cf69e642ae6d95b0e7fe5bf620aac428), and the fix is available in both bs's v1 and latest image.
